### PR TITLE
Offload some more waveform processing onto a worker

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/19756

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Offload some more waveform processing onto a worker ([\#9223](https://github.com/matrix-org/matrix-react-sdk/pull/9223)). Fixes vector-im/element-web#19756.

### CODE REVIEW

File: src/WorkerManager.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
```
Comment (alunturner) [inline src/WorkerManager.ts:2]: Super minor - think this should be 2023. Also same goes for `playback.worker.ts` and `worker.ts`.

File: src/audio/Playback.ts [hunk 2]
```diff
@@ -39,43 +44,25 @@ export interface PlaybackInterface {
...
-export interface PlaybackInterface {
-    readonly currentState: PlaybackState;
-    readonly liveData: SimpleObservable<number[]>;
-    readonly timeSeconds: number;
-    readonly durationSeconds: number;
-    skipTo(timeSeconds: number): Promise<void>;
-}
-
-export class Playback extends EventEmitter implements IDestroyable, PlaybackInterface {
+export class Playback extends EventEmitter implements IDestroyable {
```
Comment (alunturner) [inline src/audio/Playback.ts:49]: Do we still need to implement the `PlaybackInterface` here? I can't see the reason for it being removed

**Review by alunturner — COMMENTED** [review]
Looks good to me, only one very minor point (in a couple of places) and one question on an interface that isn't implemented after the changes

File: src/WorkerManager.ts (hunk 1 above)
Comment (t3chguy) [reply src/WorkerManager.ts:2]: The code was written in 2022, the PR was just a draft for ages

**Review by alunturner — APPROVED** [review]

**Pull request merged**